solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -35,6 +35,11 @@ pub const WSOL_ADDRESS: &str = "So11111111111111111111111111111111111111112";
 pub const MINIMUM_FEE_ACCOUNT_BALANCE: u64 =
     solana_program::native_token::LAMPORTS_PER_SOL;
 
+pub const BRIDGE_ESCROW_PROGRAM_ID: &str =
+    "AhfoGVmS19tvkEG2hBuZJ1D6qYEjyFmXZ1qPoFD6H4Mj";
+pub const HOOK_TOKEN_ADDRESS: &str =
+    "0x36dd1bfe89d409f869fabbe72c3cf72ea8b460f6";
```

Comment:
```suggestion
pub const BRIDGE_ESCROW_PROGRAM_ID: Pubkey =
    solana_program::pubkey!("AhfoGVmS19tvkEG2hBuZJ1D6qYEjyFmXZ1qPoFD6H4Mj");
pub const HOOK_TOKEN_ADDRESS: Pubkey =
    solana_program::pubkey!("0x36dd1bfe89d409f869fabbe72c3cf72ea8b460f6");
```

Reply:
The hook token address is ethereum address which is 20 bytes so i think that should be string only

Reply:
Yes, I realised it later on.  Ended up changing only the program ID.